documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -3473,6 +3473,55 @@ List<Poem> result = session.createNativeQuery( nativeQuery ).addEntity( Poem.cla
 ----
 ====
 
+[[ogm-mongodb-stored-proc-native]]
+
+==== Server-side JavaScript and stored procedures
+
+In MongoDB, it is possible to call server side javascripts as if it is a stored procedure.
+You can use the existing methods in JPA:
+
+.Calling server side javascript with Hibernate OGM with positional parameters
+====
+[source, JAVA]
+----
+  @Entity
+  public class Car {
+    @Id
+    private Integer id;
+
+    private String brand;
+
+    ...
+  }
+
+  EntityManager em = ...
+  StoredProcedureQuery storedProcedureQuery = em.createStoredProcedureQuery( "findMostExpensiveCars", Car.class );
```

Comment:
Same here about the consistency remark below (sorry I reviewed the main doc first so the comments look a bit backward). Let's name the stored procedures either with underscores or camel case. (Mentioning it here so that it's also fixed if you choose the underscores).